GetNedFolders.cmake

```diff
@@ -1,4 +1,4 @@
-function(_get_ned_folders _target _property _base_dir _output)
+function(__get_ned_folders _target _property _base_dir _output)
```

Comment:
Never had this bug before. Is this a well-known issue?

Reply:
That is very strange because also official CMake modules such as *FindBoost* use a single underscore as the prefix for internal function names. Do you have any hints on how to reproduce this problem? Does this happen only on Windows machines? Any specific CMake version?

Reply:
CMake Version 3.20 - it is at least happening with my environment, Windows, VSCode etc. - I suggest we implement the workaround and I will open a report at CMake, maybe there is more behind it.

Reply:
I came up with the following test, basically reflecting what we are doing:
```
cmake_minimum_required(VERSION 3.20)

project(top)

function(_my_func CALL_MYSELF)
    message(STATUS "_my_func")

    if (CALL_MYSELF)
        _my_func(OFF)
    endif()
endfunction()

function(my_func)
    message(STATUS "my_func")
    _my_func(ON)
endfunction()

my_func()
```
Tested now on three PCs with the latest CMake (2x Windows, 1x MacOs) - it worked. Really don't understand why exactly this is not working when I remove the added underscore. Once again the error message I get when executing the original code:
```
[cmake] CMake Error at cmake/omnet/GetNedFolders.cmake:31 (_get_ned_folders):
[cmake]   Maximum recursion depth of 1000 exceeded
[cmake] Call Stack (most recent call first):
[cmake]   cmake/omnet/GetNedFolders.cmake:31 (_get_ned_folders)
```

Reply:
Renaming `_get_ned_folders` to `_get_ned_folders_plumbing` should work as well, does it? I would prefer this solution over double underscores.